sdk/formrecognizer/Azure.AI.FormRecognizer/src/BoundingBox.cs

```diff
@@ -32,9 +34,10 @@ internal BoundingBox(IReadOnlyList<float> boundingBox)
         internal PointF[] Points { get; }
 
         /// <summary>
+        /// Gets one of the points that set the limits of this <see cref="BoundingBox"/>.
```

Comment:
Is it worth mentioning the order of the points? Is the order returned by the service always the same (top-left, top-right, bottom-left, bottom-right, for instance)?

Reply:
Yes, that is correct.  Would be great to call that out here!